### DISCUSSION TITLE
Beginning contribution

### DISCUSSION THREAD

**Discussion by JesperKawuma** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#32] [Outreachy 2022] [answered]
Hello guys, I am stranded at the moment. Can  someone shortly clarify these few questions because I am new here;
  Are there no open issues to begin working on?
  **Do we have a design for how the website will look plus all its functionalities?
  I can't get access to a lot of you, which I should. Are all discussions handled here on GitHub?**
 Please **@BatoolMM**, can you help me with anything useful I must be missing? Thanks

**Comment by yinkabasit** [comment]
I 

> Hello guys, I am stranded at the moment. Can someone shortly clarify these few questions because I am new here; Are there no open issues to begin working on? **Do we have a design for how the website will look plus all its functionalities? I can't get access to a lot of you, which I should. Are all discussions handled here on GitHub?** Please **@BatoolMM**, can you help me with anything useful I must be missing? Thanks

I am also confused on how to contribute to this project. I hope to get a response soon.

**Reply by JesperKawuma** [reply]
It seems nothing much is to be done here, that's why they seem not to be so active. Are you also an Outrechy applicant

**Comment by yinkabasit** [comment]
Yes, I am one of the applicant of of Outreachy. And I wish to contribute to
the OSCSA project.

**Comment by BatoolMM** [comment, marked as answer]
Welcome, @JesperKawuma and @yinkabasit to the community!

We have two projects for Outreachy, you can find them here:
1- [Improve the bilinguality and accessibility of the website](https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website/issues/12)
2- Improve Localisation for [Open Educational Resources](https://github.com/Open-Science-Community-Saudi-Arabia/localisation/issues/13).

Some of the tasks in the issue have been addressed - you are also welcome to review open PR!

Let me know if you are still confused!

Feel free to comment under the issue, if you have any questions!
